tests/attr/helpers/test_config.py

```diff
@@ -20,7 +20,9 @@
 from captum.attr._core.layer.layer_gradient_shap import LayerGradientShap
 from captum.attr._core.layer.layer_gradient_x_activation import LayerGradientXActivation
 from captum.attr._core.layer.layer_integrated_gradients import LayerIntegratedGradients
+from captum.attr._core.layer.layer_lrp import LayerLRP
 from captum.attr._core.lime import Lime
+from captum.attr._core.lrp import LRP
```

Comment:
Are all configurations here being run by data parallel test ?

Reply:
Yes, each config is run as a data parallel test